### ISSUE TITLE
org-mode pretty entities within words not rendered correctly

### ISSUE THREAD

**Issue by santoshs** [gohugoio/hugo#9824]
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
Hugo Static Site Generator v0.80.0/extended linux/amd64 BuildDate: unknown
</pre>

### Does this issue reproduce with the latest release?
Yes this gets reproduced in the latest

```
$ ~/Downloads/hugo_0.97.3_Linux-64bit/hugo version
hugo v0.97.3-078053a43d746a26aa3d48cf1ec7122ae78a9bb4 linux/amd64 BuildDate=2022-04-18T17:22:19Z VendorInfo=gohugoio
```

The org-more pretty entities are correctly rendered, but when they are as part of a word (as mentioned in the [manual](https://orgmode.org/manual/Special-Symbols.html#Special-Symbols)), the `{}` are displayed as part of the rendered output.

For example

```
The time is in \micro{}S
```

Will render as

```
The time is in µ{}S
```

**Comment by kaushalmodi** [comment]
Can you open this issue in the Org parsing library used by Hugo: https://github.com/niklasfasching/go-org?

You can then link that issue here.

**Comment by jmooring** [comment]
Work around: omit the curly brackets.

```
The time is in \microS
```

**Comment by kaushalmodi** [comment]
@jmooring I wouldn't recommend that workaround. While that might make things work with Hugo, it won't work in Emacs and other Org exports.

Below, notice that `\microS` won't render the mu symbol in-buffer.

Also, when that Org snippet is exported using one of the built-in exporters, `\microS` won't export correctly. The `{}` is needed to separate the special Org entity from the follow text.. it's kind of what `${foo}bar` vs `$foobar` does in bash.

**Comment by kaushalmodi** [comment]
@santoshs Another suggestion for workaround.

> Will render as
> ```
>  The time is in µ{}S
> ```

You can post-process the `.RawContent` and replace `(\\[a-z]+)\{\}` regex with `${1}` in your Hugo templates.

**Comment by kaushalmodi** [comment]
heyy.. @jmooring Are you an Emacs user too?

**Comment by jmooring** [comment]
@kaushalmodi Only when trying troubleshoot reported issues.

**Comment by jmooring** [comment]
Re-opening. This is a bug, and needs to be addressed upstream. I'll take care of that.

**Comment by kaushalmodi** [comment]
@santoshs 

> but the in buffer rendering breaks. But I can live with that :-)

/groan

You are polluting your Org source! That workaround will result in incorrect pdf, HTML and other exports too!

But ..  ¯\\\_(ツ)_/¯

**Comment by kaushalmodi** [comment]
@santoshs I would rather do this in my Hugo template:

```
        {{ .Content | replaceRE "µ{}" "µ" | safeHTML }}
```

**Comment by jmooring** [comment]
@kaushalmodi 

How should `\microS` be rendered?

**Comment by kaushalmodi** [comment]
@santoshs 

An even better workaround

```
        {{ .RawContent | replaceRE `(\\[a-zA-Z]+)\{\}` "${1}" | $.RenderString }}
```

---

@jmooring 

> How should \microS be rendered?

It would render literally as `\microS` (as in the screenshot in https://github.com/gohugoio/hugo/issues/9824#issuecomment-1109853561).

**Comment by jmooring** [comment]
@kaushalmodi Thanks. Please check the upstream issue for accuracy.

https://github.com/niklasfasching/go-org/issues/79

**Comment by jmooring** [comment]
This was resolved in v0.101.0 with https://github.com/gohugoio/hugo/commit/35fa192838ecfa244335fca957e55d3956a48665
